xskillscore/tests/test_contingency.py

```diff
@@ -1,49 +1,61 @@
 import numpy as np
 import pytest
 import xarray as xr
-
 from xskillscore import Contingency
+from sklearn.metrics import confusion_matrix
+
+
+DIMS = [['time'],['lon'],['lat'],'time',['lon','lat','time']]
```

Comment:
Maybe a tuple? I also think `black` must not have been run here because there should be spacing between the list element.s